config.lua.dist

```diff
@@ -175,6 +175,7 @@ deathLosePercent = -1
 housePriceEachSQM = 1000
 houseRentPeriod = "never"
 houseOwnedByAccount = false
+houseBuyLevel = 100 -- if you want disable, just put 0
```

Comment:
-- Houses
-- NOTE: set housePriceEachSQM to -1 to disable the ingame buy house functionality
-- NOTE: set houseBuyLevel to if you want disable, just put 0
-- Periods: daily/weekly/monthly/yearly/never
housePriceEachSQM = 1000
houseRentPeriod = "never"
houseOwnedByAccount = false
houseBuyLevel = 100

Reply:
corrigi.